create/faq.md

```diff
@@ -189,4 +176,45 @@ See our [contact page](/learn/what_is_meilisearch/contact.md).
 
 ## I have just updated MeiliSearch, and I am getting an error: "Cannot open database, expected MeiliSearch engine version..."
 
-MeiliSearch minor versions are not compatible with each other because the way we represent data internally changes with the addition of new breaking features. To fix this, it suffices to delete your database folder (`data.ms` by default) and re-index your documents with the new version.
+MeiliSearch minor versions are not compatible with each other because the way we represent data internally changes with the addition of new breaking features. To fix this, it suffices to delete your database folder (`data.ms` by default) and re-index your documents with the new version. See [updating MeiliSearch](/reference/features/installation.md#updating-meilisearch) for more information.
+
+## What are the recommended requirements for hosting a MeiliSearch instance?
+
+**Short answer**: It's impossible to say. We recommend [experimenting with RAM availability and database size](#how-does-hardware-affect-search-speed) to find the hardware + relevancy setup that works best for your dataset.
+
+**Long answer**: The disk size and RAM usage of your MeiliSearch database can vary widely. The following factors have a great effect on space and memory use:
+
+- The number of documents
+- The size of documents
+- The number of indexed fields
+- The number of faceted fields
+- The size of each update
+- The number of different words present in documents (beware heavily multi-lingual datasets and fields with unique words, such as IDs)
+
+All of this means that **it's almost impossible to estimate the size and memory usage of a MeiliSearch database** before creation, even based on the size of its documents. There are, however, a few [general principles](#how-does-hardware-affect-search-speed) you should know.
+
+## How does hardware affect search speed?
+
+Because MeiliSearch uses a [memory map](/reference/under_the_hood/storage.md#lmdb), **search speed is based on the ratio between RAM and database size**. In other words:
+
+- A big database + a small amount of RAM => slow search
+- A small database + tons of RAM => lightning fast search
+
+The number of CPU cores has no impact on index speed (indexing is single-threaded... for now 😉). However, **the more cores you provide to the engine, the more search queries it will be able to process at the same time**.
+
+### Speeding Up MeiliSearch
+
+MeiliSearch is designed to be fast (≤50ms response time), so speeding it up is rarely necessary. However, if you find that your MeiliSearch instance is querying slowly, there are two primary methods to speed it up:
```

Comment:
It sounds marketing, but `≤50ms response time` means nothing. Is it always that fast no matter the number of documents or the type of queries? Maybe we should be more precise on this stats, don't want it to fuck us up in the future :)

Reply:
Apparently the 50ms number actually has some basis in the code. Will check w/ core for more details :)